tests/functional/codegen/features/test_transient.py

```diff
@@ -55,3 +44,316 @@ def setter(k: address, v: uint256):
 
```

Comment:
we should probably add something to this file which explains that, with eth-tester, each call happens in an isolated transaction and so we need to test get/set within a single contract call. (we should remove this restriction in the future by migrating away from eth-tester).